timetracker/timetracker/urls.py

```diff
@@ -16,11 +16,47 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.http import HttpResponse
+from django.views.decorators.http import require_http_methods
 
+# If there's no name, display a form asking for name.
+# This is the HTML for that form.
+form_page_html = """
+  <html>
+    <body>
+      <p>Hi, what's your name?</p>
 
+      <form method="GET">
```

Comment:
Good point!

I saw Bellana's PR, nice one. I have to get @funkybob to apologise for me tonight – I forgot to record the AUDIO for the last lesson. 3.5GBs of screen recording with no audio in a little useless :(